yarn-project/pxe/src/note_processor/produce_note_dao.ts

```diff
@@ -81,6 +79,8 @@ async function findNoteIndexAndNullifier(
   let siloedNoteHash: Fr | undefined;
   let uniqueSiloedNoteHash: Fr | undefined;
   let innerNullifier: Fr | undefined;
+  const txNullifier = Fr.fromBuffer(txHash.toBuffer());
```

Comment:
Would prefer calling this a firstNullifier because we use that name in other places but it's just a nit.